docs/standard/microservices-architecture/implement-resilient-applications/monitor-app-health.md

```diff
@@ -276,11 +276,11 @@ Finally, if you're storing all the event streams, you can use Microsoft Power BI
     [https://docs.microsoft.com/azure/service-fabric/service-fabric-health-introduction](/azure/service-fabric/service-fabric-health-introduction)
 
 -   **Azure Application Insights**
-    [https://azure.microsoft.com/services/application-insights/](https://azure.microsoft.com/services/application-insights/)
+    <https://azure.microsoft.com/services/application-insights/>
```

Comment:
@mvelosop @CESARDELATORRE this link now redirects to Azure Monitor. Probably a new name?